documentation/assets/measured.svg

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
As a nice to have we could have this SVG optimized with https://jakearchibald.github.io/svgomg/ to get it from 700 bytes to 422 bytes. Honestly, probably not even worth it but my eyes now always spot unoptimized SVGs after working on webperf for a while 🤣

Reply:
I will go ahead and do this, this svg is straight from wikimedia.

- [ ] Optimize svg.